### DISCUSSION TITLE
[BP#52] Donner des fonctions en paramètre à setTimeout() et setInterval() plutôt que des string

### DISCUSSION THREAD

**Discussion by ddebarbieux** [cnumr/best-practices#153] [Bonnes pratiques] [answered]
La description de cette bonne pratique (_D'aprés la bonne pratique précédente_) donne envie de la supprimer et de l'intégrer dans la BP#51.

**Comment by ddebarbieux** [comment]
[Et la BP#51 va peut être surprimée.](https://github.com/cnumr/best-practices/discussions/61).

**Reply by ACTLEM** [reply]
La suppression de la bonne pratique 51 a effectivement été actée.

**Comment by tbroyer** [comment]
Du coup, on conserve cette BP pour tout ce qui est "eval" ? ou on la supprime comme la 51 ?

**Reply by ACTLEM** [reply]
J'ai fait quelques tests et  je ne vois pas de différence, donc j'ai tendance à dire qu'elle peut être supprimée sauf si quelqu'un me démontre l'inverse de ce que je constate.

**Comment by ACTLEM** [comment, marked as answer]
Note équipe projet: Pour supprimer étant les résultats de tests effectuées.

Issue créée: https://github.com/cnumr/best-practices/issues/169
